apps/bare-expo/android/gradle.properties

```
@@ -10,15 +10,13 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx512m -XX:MaxMetaspaceSize=256m
-org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m
+org.gradle.jvmargs=-Xmx20240M -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

- In the deleted line below it was `-Xmx10240M`:
   ```suggestion
   org.gradle.jvmargs=-Xmx10240M -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
   ```
- What about deleted `-XX:MaxMetaspaceSize=512m`? Was it unused because it was overridden below?

---

`-Xmx` - I increased that from 1 to 2 GB. 
`XX:MaxMetaspaceSize` - Should be present. I revert that.